client/src/components/Portfolio.tsx

```diff
@@ -0,0 +1,69 @@
+import Navbar from "./Navbar"
+import styled from "styled-components"
+import { ReactNode } from "react"
+import {Button, Container, makeStyles, Typography} from "@material-ui/core"
+
+const useStyles = makeStyles(theme => ({
+    container: {
+        marginLeft: theme.spacing(10),
+        marginRight: theme.spacing(10),
+        marginTop: theme.spacing(0),
+        display: "flex",
+        flexDirection: "column"
+    },
+    pricesContainer: {
+       display: "flex",
+       flexDirection: "row"
+    },
+    changeComponent: {
+        background: "#00FF00",
+        color: "white",
+        fontWeight: "bold",
+        marginLeft: theme.spacing(20)
+    },
+    changeValueComponent: {
+        color: "#00FF00"
+    }
+}))
+
+function numberWithCommas(valueStr: string) {
+    return valueStr.replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+}
+
+function calculatePortfolioValue(cryptos: any[]): number {
```

Comment:
Might want to add a TODO for the `any[]` here, so we can find it later.

Reply:
used <Box> for flex box features that are needed.